The pipeline is blocked bc of two trivial linter errors:

```
homeassistant/components/velux/cover.py:63:80: E501 line too long (81 > 79 characters)
homeassistant/components/velux/cover.py:63:82: W291 trailing whitespace
```

@gibman : may you change them?

--- a/homeassistant/components/velux/cover.py
+++ b/homeassistant/components/velux/cover.py
@@ -60,7 +60,7 @@ class VeluxCover(CoverDevice):
 
     @property
     def device_class(self):
-        """Define this cover as either window/blind/awning/shutter"""
+        """Define this cover as either window/blind/awning/shutter."""
         from pyvlx.opening_device import Blind, RollerShutter, Window, Awning
         if isinstance(self.node, Window):
             return 'window'

--- a/homeassistant/components/velux/cover.py
+++ b/homeassistant/components/velux/cover.py
@@ -60,7 +60,7 @@ class VeluxCover(CoverDevice):
 
     @property
     def device_class(self):
-        """Define this cover as either window, blind, awning or rollershutter.""" 
+        """Define this cover as either window/blind/awning/shutter"""
         from pyvlx.opening_device import Blind, RollerShutter, Window, Awning
         if isinstance(self.node, Window):
             return 'window'

